docs/designs/conceptual-versioning.md

```diff
@@ -0,0 +1,395 @@
+# Conceptual versioning - Phase 1
```

Comment:
what's the relationship between versioning and conceptual-versioning document?

Reply:
versioning contains `conceptual versioning`, `reference versioning`

Reply:
we should consider to combine them into one design?

Reply:
we haven't designed it yet.

---
In reply to: [227213956](https://github.com/dotnet/docfx/pull/3522#discussion_r227213956) [](ancestors = 227213956)

Reply:
we should combine conceptual versioning and reference versioning.